File: packages/@monomind/monograph/__tests__/fixtures/sample.kt
import com.example.model.User
import com.example.service.Logger

interface UserService {
    fun getUser(id: Int): User?
    fun createUser(name: String): Boolean
}

data class User(val id: Int, val name: String)

class UserServiceImpl : UserService {
    private val users = mutableListOf<User>()

    override fun getUser(id: Int): User? {
        return users.find { it.id == id }
    }

    override fun createUser(name: String): Boolean {
        users.add(User(users.size + 1, name))
        return true
    }

    private fun helperFn(s: String): String {
        return s.uppercase()
    }
}

fun topLevelFn(x: Int): Int {
    return x * 2
}

File: packages/@monomind/monograph/__tests__/fixtures/sample.py
from os import path

class UserService:
    def __init__(self, db):
        self.db = db

    def get_user(self, user_id: str):
        return self.db.find(user_id)

def helper_fn(x: int) -> int:
    return x * 2

def _private_fn(y: int) -> int:
    return y + 1
